appinventor/appengine/src/com/google/appinventor/client/explorer/project/ProjectManager.java

```diff
@@ -12,13 +12,10 @@
 import com.google.appinventor.shared.rpc.project.ProjectNode;
 import com.google.appinventor.shared.rpc.project.UserProject;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
You should revert this change, we do not want exports of the form foo.*, but instead explicit exports of modules.

Reply:
Will fix this in next commit, had my IDE misconfigured so it automatically did a star import.